docker-entrypoint.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -eu
+
+# If the first argument looks like a flag, assume we want to run changedetection
```

Comment:
hmmm what is the problem you're trying to solve here?

Reply:
Assuming you're referring to the line below: If the container is e.g. run with `docker-compose run changedetection -c`, this would run `python ./changedetection.py -d datastore -c` inside the container. It's mostly a convention, nothing that's necessary, just nice to have. If you prefer it removed, no worries. :)